test/create_db.sh

```diff
@@ -78,7 +78,7 @@ for db in $DBS; do
     # sql-migrate will default to ./dbconfig.yml and treat all configured dirs
     # as relative.
     cd "${dbpath}"
-    r=`sql-migrate up -env="${dbname}" | xargs echo`
+    r=`sql-migrate up -env="${dbname}" | xargs -0 echo`
```

Comment:
This is surprising! Does sql-migrate send 0-separated items by default?

Reply:
It's confusing to me too, frankly. I don't think it does. But it frequently prints out messages that contain the single quote character `'` (e.g. in contractions like `can't`), and xargs immediately barfs on those:
```sh
$ echo "can't do this" | xargs echo
xargs: unmatched single quote; by default quotes are special to xargs unless you use the -0 option
```

I kept running into this while debugging my sql schema, and adding the -0 caused things to work.

Reply:
```
       -0, --null
              Input items are terminated by a null character instead of by whitespace, and the quotes and backslash are not special (every character is tak‐
              en literally).  Disables the end of file string, which is treated like any other argument.  Useful when input items might contain white space,
              quote marks, or backslashes.  The GNU find -print0 option produces input suitable for this mode```